cmake/projects/sparsehash/hunter.cmake

```diff
@@ -19,17 +18,13 @@ hunter_add_version(
   VERSION
   "2.0.2"
```

Comment:
I think it make sense to show here that version is not clean and was patches, e.g. use `v2.0.2-hunter` instead of `2.0.2`.